### ISSUE TITLE
static initialization of vpn_config

### ISSUE THREAD

**Issue by DimitriPapadopoulos** [adrienverge/openfortivpn#207]
The `vpn_config`structure is initialized in _main.c_ using the `init_vpn_config` macro defined in _config.h_.

While it is nice to have the initialization code in a macro function just after the definition of the structure in _config.h_, we could use static struct initialization in _main.c_ instead of `memset` + the macro function call. Especially since `vpn_config` is initialized only once in _main.c_.

Same goes for the `destroy_vpn_config` macro.

**Comment by DimitriPapadopoulos** [comment]
@adrienverge Would you agree on static initialization of `cfg` instead of `memset()` + macro initialization function? The downside is that initialization wouldn't happen in _config.h_ where the `cfg` is defined, but in _main.c_.

**Comment by adrienverge** [comment]
Yes I agree, I don't see any issue it could cause!

**Comment by mrbaseman** [comment]
it looks good to me, but I would first try to land the other pull requests and resolve the upcoming conflicts in this one, rather than changing this central place of the code and having to fight with the conflicts in most other pull requests

**Comment by DimitriPapadopoulos** [comment]
Indeed. I'll keep this last!
